TESTS/block_device/qspif/main.cpp

```diff
@@ -0,0 +1,339 @@
+#include "mbed.h"
```

Comment:
Is this needed if use the mbed namespace?

Reply:
I checked again, both are not required - I'll remove